From b6bae49296057efcb93f8c30dadad111eff0d1e5 Mon Sep 17 00:00:00 2001
From: Eric Van Norman <ericvn@us.ibm.com>
Date: Mon, 15 Nov 2021 12:19:40 -0600
Subject: Fix docker.save having `-default` in tar name (#36061)

* Fix docker.save having `-default` in tar name

* Remove extra change copying tar file
---
 tools/docker-builder/main.go | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tools/docker-builder/main.go b/tools/docker-builder/main.go
index 24c8c889cf..ff33dcfbdc 100644
--- a/tools/docker-builder/main.go
+++ b/tools/docker-builder/main.go
@@ -246,7 +246,7 @@ func ConstructBakeFile(a Args) (map[string]string, error) {
 				t.Outputs = []string{"type=registry"}
 			} else if args.Save {
 				n := target
-				if variant != "" {
+				if variant != "" && variant != DefaultVariant { // For default variant, we do not add it.
 					n += "-" + variant
 				}
 
-- 
2.35.3

